text-ui-test/Deadline.java

```diff
@@ -0,0 +1,15 @@
+import java.util.*;
```

Comment:
I'm not sure why you have classes under your `text-ui-test` folder. I suppose these are outdated.